index.js

```diff
@@ -28,6 +28,13 @@
 }(this, function () {
   "use strict";
 
+  // Use classList if available, else fall back to manual string comparison
+  var hasClass = document.body.classList
+    ? function(el, className) { return el.classList.contains(className); } 
+    : function(el, className) {
+      return (' ' + el.className + ' ').indexOf(' ' + className + ' ') > -1;
```

Comment:
but what's the spread? I trust that value, but don't trust that it matters for people who can run React apps at all.